### DISCUSSION TITLE
Roadmap meeting notes

### DISCUSSION THREAD

**Discussion by jhfnetboy** [savour-labs/savour-docs-chinese#9] [General]
Here we talking about the milestone, roadmap and future of Savour.

**Comment by jhfnetboy** [comment]
Quick self-introduction：

Jason, an Ethereum community build, would build Ethereum in the left life. Fucus on Layer3 in the long term.

**Reply by guoshijiang** [reply]
welcome

**Reply by jhfnetboy** [reply]
11-26，biweekly meeting
Talking about the future step

**Comment by guoshijiang** [comment]
For us, I think the first thing we should finish multi staking protocol.
